Dockerfile-debian.template

```diff
@@ -37,19 +37,13 @@ RUN set -eux; \
 ENV DART_SDK /usr/lib/dart
 ENV PATH $DART_SDK/bin:$PATH
 
-ENV PLATFORM       {{PLATFORM}}
-ENV ARCH           {{ARCH}}
-ENV CHANNEL        {{CHANNEL}}
-ENV DART_VERSION   {{DART_VERSION}}
-ENV DART_SHA256    {{DART_SHA256}}
-
 WORKDIR /root
 RUN set -eux; \
-    SDK="dartsdk-$PLATFORM-$ARCH-release.zip"; \
+    SDK="dartsdk-linux-$(dpkg --print-architecture)-release.zip"; \
```

Comment:
This won't work, it returns amd64 instead of x64.

Reply:
Just saw that, thanks. On it.